cmd/deploy/deploy.go

```diff
@@ -297,11 +312,19 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		fmt.Sprintf("%s=%s", model.OktetoNamespaceEnvVar, okteto.Context().Namespace),
 	)
 
-	for _, command := range opts.Manifest.Deploy.Commands {
-		if err := dc.executor.Execute(command, opts.Variables); err != nil {
-			log.Infof("error executing command '%s': %s", command, err.Error())
-			return fmt.Errorf("error executing command '%s': %s", command, err.Error())
-		}
+	err = dc.deploy(opts)
+	if err != nil {
+		output = fmt.Sprintf("%s\nApp '%s' deployment failed: %s", output, opts.Name, err.Error())
+		cfg = app.SetStatus(cfg, app.ErrorStatus, output)
+	} else {
+		output = fmt.Sprintf("%s\nApp '%s' successfully deployed", output, opts.Name)
+		cfg = app.SetStatus(cfg, app.DeployedStatus, output)
+	}
+	if err := configmaps.Deploy(ctx, cfg, opts.Manifest.Namespace, c); err != nil {
```

Comment:
We should also add the output of the command, git repo and branch to have the same experience as deploying from the UI

Reply:
+1

Reply:
What if it's a folder without `.git`?

Reply:
then, we don't add the git repo and branch data